doc/user/spatial_block_parameters.rst

```diff
@@ -0,0 +1,329 @@
+************************
+Spatial block parameters
+************************
+
+Many parameters assigned on a ``Block`` are scalar quantities that are useful for visualization
+and simple queries (e.g., block with the maximum burnup in an assembly). Spatial parameters in
+the a block, such as power produced by each pin, is also of interest. Especially when communicating
+data to physics codes that support sub-block geometric modeling. This page will talk about
+how spatial information is assigned to components on a block, how spatial data can be assigned
+and accessed, and how those data may or may not be updated by the framework.
+
+Sub-block spatial grid
+======================
+
+There are two ways to create the block grid: explicitly via blueprints, or via an automated builder. The former
```

Comment:
```suggestion
There are two ways to create the block grid: explicitly via blueprints or via an automated builder. The former
```

Reply:
Commas usually only separate items if the list is longer than 2 items long.

I won't die on this hill.